scalability_and_performance/ztp_far_edge/ztp-sno-additional-worker-node.adoc

```diff
@@ -0,0 +1,26 @@
+:_content-type: ASSEMBLY
+[id="ztp-sno-additional-worker-node"]
+= Additional worker nodes in a {sno} cluster
+include::_attributes/common-attributes.adoc[]
+:context: sno-additional-worker
+
+toc::[]
+
+You can add one or more worker nodes to an existing {sno-caps} (SNO) cluster to increase CPU resources used by the original SNO control plane node. The addition of worker nodes does not require any downtime for the existing SNO.
```

Comment:
{sno-caps} variable is intended for use at the start of sentences. For the most part, just use {sno}.

```suggestion
You can add one or more worker nodes to an existing {sno} cluster to increase CPU resources available to the control plane node. The addition of worker nodes does not require any downtime for the existing existing {sno} cluster.
```